packages/web-app/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,177 @@
+import { Button, Text } from '@saladtechnologies/garden-components'
+import classNames from 'classnames'
+import type CSS from 'csstype'
+import type { FunctionComponent } from 'react'
+import type { WithStyles } from 'react-jss'
+import withStyles from 'react-jss'
+import { DefaultTheme, type SaladTheme } from '../../SaladTheme'
+
+const styles: (theme: SaladTheme) => Record<string, CSS.Properties> = (theme: SaladTheme) => ({
+  paginationWrapper: {
+    width: '100%',
+    color: theme.lightGreen,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    flexDirection: 'row',
+    gap: '16px',
+    padding: '16px',
+    paddingLeft: '12px',
+  },
+  navigationButtonWrapper: {
+    width: '100px',
+  },
+  pagesNavigationWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    gap: '6px',
+    width: '250px',
+    paddingLeft: '12px',
+    paddingRight: '12px',
+  },
+  centralPagesWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    gap: '6px',
+  },
+  edgePagesWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    gap: '10px',
+    width: '65px',
+    paddingLeft: '4px',
+    paddingRight: '4px',
+  },
+  currentPageText: {
+    color: theme.darkBlue,
+  },
+  button: {
+    minWidth: '32px',
+    padding: '0px 6px',
+    height: '32px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    backgroundColor: 'transparent',
+    color: 'inherit',
+    border: `1px solid ${theme.lightGreen}`,
+    font: 'inherit',
+    cursor: 'pointer',
+    outline: 'inherit',
+    fontFamily: theme.fontMallory,
+  },
+  currentPageButton: {
+    backgroundColor: theme.lightGreen,
+    color: theme.darkBlue,
+    fontWeight: 500,
+  },
+  disabled: {
+    opacity: 0.5,
+  },
+})
+
+interface Props extends WithStyles<typeof styles> {
+  itemsTotalAmount: number
+  itemsPerPageAmount: number
+  currentPageNumber?: number
+  onPageChange: (pageNumber: number) => void
+}
+
+const _Pagination: FunctionComponent<Props> = ({
+  classes,
+  itemsTotalAmount,
+  itemsPerPageAmount = 10,
+  currentPageNumber = 1,
+  onPageChange,
+}) => {
+  const totalPagesAmount = Math.ceil(itemsTotalAmount / itemsPerPageAmount)
+  const isPreviousPageEnabled = currentPageNumber !== 1
```

Comment:
```suggestion
  const isPreviousPageEnabled = currentPageNumber > 1
```
Using > allows for more flexibility. If `currentPageNumber` becomes less than 1 (due to a bug or unexpected input), the condition would still correctly disable the previous page button.

Reply:
don't we hide the bug this way ?